gradle.properties

```diff
@@ -1,3 +1,4 @@
 org.gradle.parallel=true
 org.gradle.caching=true
-org.gradle.jvmargs=-Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx2048m -Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8
+android.useAndroidX=true
```

Comment:
the `android.useAndroidX=true` flag was removed from all the github actions and placed here to avoid duplication.